app/src/main/java/vn/edu/usth/minigh/fragments/TreeFragment.kt

```diff
@@ -0,0 +1,55 @@
+package vn.edu.usth.minigh.fragments
+
+import kotlinx.coroutines.launch
+
+import android.os.Bundle
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.LinearLayout.LayoutParams
+import android.widget.LinearLayout.LayoutParams.MATCH_PARENT
+import android.widget.LinearLayout.LayoutParams.WRAP_CONTENT
+import android.widget.TextView
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.add
+import androidx.fragment.app.commit
+import androidx.lifecycle.lifecycleScope
+
+import vn.edu.usth.minigh.R
+import vn.edu.usth.minigh.api.github
+
+class TreeNodeFragment : Fragment(R.layout.fragment_tree_node) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val args = requireArguments()
+        // TODO: handle symlinks and git submodules
+        val type = if (args.getString("type") == "dir") {
+            R.string.fa_folder
```

Comment:
Please use [solid folder](https://fontawesome.com/icons/folder?style=solid) for more visual distinction.

Reply:
Addressed in e1e476a.